packages/core/src/package.ts

```diff
@@ -73,10 +74,7 @@ const build = async (config: EnhancedConfig, packageName: string) => {
         const entry = entries.find(
           ({ entryPath }) => chunkInfo.facadeModuleId === entryPath,
         );
-
-        if (!entry) {
-          throw new Error('Unable to name entry file');
-        }
+        assert(entry != null, 'entry not found');
```

Comment:
Should this be `!==`?
```suggestion
        assert(entry !== null, 'entry not found');
```

Reply:
I explicitly chose `!=`.

Reply:
<img width="676" alt="error" src="https://user-images.githubusercontent.com/3297808/210485930-ab20f5a9-5aac-4fab-8cea-5b25299d3760.png">